File: csharp/hamming/Hamming.cs
using System;

public static class Hamming
{
    public static int Distance(string firstStrand, string secondStrand)
    {
        int lenFirst = firstStrand.Length;
        int lenSecond = secondStrand.Length;

    if (lenFirst != lenSecond){
        throw new ArgumentException("Cannot compare two unequal strings");
    }
        int HammingScore = 0;
        for (int i = 0; i < lenFirst; i++)
        {
            if (firstStrand[i] != secondStrand[i]){
                HammingScore += 1;
            }
        }

        return HammingScore;
    }
}

File: python/hamming/hamming.py
def distance(strand_a, strand_b):
    if not len(strand_a) == len(strand_b):
        raise ValueError("strands of unequal length")

    hamming_score = 0
    for i in range(len(strand_a)):
        if strand_a[i] != strand_b[i]:
            hamming_score += 1

    return hamming_score
